auth0/v3/authentication/base.py

```diff
@@ -14,13 +14,20 @@ def get(self, url, params=None, headers=None):
         return requests.get(url=url, params=params, headers=headers).text
 
     def _process_response(self, response):
+        error_response = response.status_code is None or response.status_code >= 400
+
         try:
-            text = json.loads(response.text) if response.text else {}
+            result = json.loads(response.text) if response.text else {}
+            error_code = result.get('error', '')
```

Comment:
I've seen some error responses return `code` instead of `error`. Both meaning the same thing. If this value is not present I'd anyway set something like `"a0.sdk.internal.unknown"`.